### PR TITLE
LPS-108804 Remove AUI from Liferay.LayoutExporter utils

### PR DESCRIPTION
**Pull request by markocikos**
Hey @jbalsas ,

There are five `LayoutExporter `utils. Four of them are not used anywhere, one is _maybe_ used.

I removed AUI from the unused ones and deprecated them. I also took the route of least changes, not rewriting them as other utils to `global.es.js`, so we can simply delete the `layout_exporter.js` file after 7.3.

The last one, `LayoutExporter.publishToLive`, is used in two places in staging, so I replaced those to usages with content of the util. I suspect that they are, in fact, not used, and those conditions when it is used in `staging-taglib` do not exist any more. So, ideal solution would probably be to remove a large part of `staging_actions.jspf`, but I am hesitant to do this unless I have confirmation from  somebody responsible and knowledgeable about product. The solution in this PR is a safer half-measure. The only downside is, I was not able to test the code live, but these are low-risk changes, and CI will add a layer of security.

Please review the code.

Thank you!

### CODE REVIEW

**Comment by markocikos** [comment]
ci:test

**Comment by jbalsas** [comment]
@ealonso, do you have any idea if these might be used anywhere outside of Staging? 

@markocikos, could you please work with @esanzp to figure out who to reach in the Staging team for confirmation?

**Comment by ealonso** [comment]
@jbalsas I think the only place where we are using it is on Staging

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/layout_exporter.js [hunk 1]
```diff
@@ -12,93 +12,69 @@
...
-				uri: options.url
-			});
-		}
-	};
-
-	Liferay.provide(
-		LayoutExporter,
-		'all',
-		options => {
+const Liferay = Liferay || {};
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/layout_exporter.js]: Is this possible? In which case is `Liferay` not defined here? This is a clear smell that we should address.

**Comment by jbalsas** [comment]
Can you forward this to @julien or @wincent so they can take a look and work together to see if we can simplify it a little bit by removing the `Liferay = Liferay || {}` declaration and the `(Liferay)` invocation?

Thanks!

**Pull request closed without merging**